### ISSUE TITLE
Deprecated dependency WiringPi

### ISSUE THREAD

**Issue by Ditman352** [PDA-UR/LagBox#2]
is the wiringPi.h file missing. I'm getting the error below.

@raspberrypi:~/LagBox/src $ make
mkdir ../bin
gcc -Wall -c inputLatencyMeasureTool.c -o ../bin/inputLatencyMeasureTool.o -std=gnu99
inputLatencyMeasureTool.c:42:10: fatal error: wiringPi.h: No such file or directory
   42 | #include <wiringPi.h>
      |          ^~~~~~~~~~~~
compilation terminated.
make: *** [Makefile:6: main] Error 1

If I try to continue I get this when I try to use gui and start measurement.

888888@raspberrypi:~/LagBox/gui $ python3 latency_gui.py
/home/888888/LagBox/gui/latency_gui.py:100: SyntaxWarning: "is not" with a literal. Did you mean "!="?
  if QtWidgets.QWizard.currentId(self) is not 1:
/home/888888/LagBox/gui/latency_gui.py:438: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(line) is 0:  # Count all empty lines
/home/888888/LagBox/gui/latency_gui.py:468: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if len(line) is 0:
/home/888888/LagBox/gui/latency_gui.py:682: SyntaxWarning: "is" with a literal. Did you mean "=="?
  elif len(line) is 0:
/home/888888/LagBox/gui/DataPlotter.py:40: SyntaxWarning: "is" with a literal. Did you mean "=="?
  if current_file[i][0] is '#':  # If row is a comment
cat /sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed
Speed: cat:'/sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed':NosuchfileordirectoryM
cat /sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed
Speed: cat:'/sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed':NosuchfileordirectoryM
cat /sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed
Speed: cat:'/sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed':NosuchfileordirectoryM
cat /sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed
Speed: cat:'/sys/devices/platform/scb/fd500000.pcie/pci0000:00/speed':NosuchfileordirectoryM
Device name: Performance Designed Products Rock Candy Wired Controller for Ni
Device type ID: 1
Values for bInterval of device:
['5', '5']
Starting Key Detection
../bin/inputLatencyMeasureTool -m 3 -tmin 100 -tmax 10000 -b 308 -d 1 -event 3 -n 1000 -name 'Performance Designed Products Rock Candy Wired Controller for Ni'
b'/bin/sh: 1: ../bin/inputLatencyMeasureTool: not found\n'
b''
QThread: Destroyed while thread is still running
Aborted

**Comment by A-Schmid** [comment]
Hi!
wiringPi is an external dependency you can find all information about it here: http://wiringpi.com/
You have to install it in order to build the the "inputLatencyMeasureTool".
In your case, the (Python-) GUI crashed because the actual measuring tool (written in C) could not be found, as it did not get built because of the missing dependency.
Hope I could help :)

**Comment by Ditman352** [comment]
Yes this was the issue. However "http://wiringpi.com/" didn't work for me and I did need to install wiringPi.h from a 3rd party source. https://github.com/WiringPi/WiringPi got me to the next step. Still working....

**Comment by Ditman352** [comment]
Updating the page below would be nice once this gets worked out. 

https://hci.ur.de/projects/latency/howto
